contracts/ERC7432/ERC7432WrapperForERC4907.sol

```diff
@@ -0,0 +1,250 @@
+// SPDX-License-Identifier: CC0-1.0
+
+pragma solidity 0.8.9;
+
+import { IERC721 } from '@openzeppelin/contracts/token/ERC721/IERC721.sol';
+import { IERC721Receiver } from '@openzeppelin/contracts/token/ERC721/IERC721Receiver.sol';
+import { ERC721Holder } from '@openzeppelin/contracts/token/ERC721/utils/ERC721Holder.sol';
+import { IERC7432 } from '../interfaces/IERC7432.sol';
+import { IERC4907 } from '../interfaces/IERC4907.sol';
+import { IERC7432VaultExtension } from '../interfaces/IERC7432VaultExtension.sol';
+import { IOriumWrapperManager } from '../interfaces/IOriumWrapperManager.sol';
+import { IWrapNFT } from '../interfaces/DoubleProtocol/IWrapNFT.sol';
+
+contract ERC7432WrapperForERC4907 is IERC7432, IERC7432VaultExtension, ERC721Holder {
```

Comment:
Maybe add the `Pausable` modifiers to functions?

Reply:
We can pause the contract by using the map and max duration functions of the manager contract (we can pause individual collections there)